src/handlers/major_change.rs

```diff
@@ -127,11 +127,18 @@ async fn handle_input(
                 cmnt.post(&ctx.github).await?;
                 return Ok(());
             }
-            (format!(
-                "A new proposal has been announced: [#{}]({}). It will be brought up at the next meeting.",
-                issue.number,
-                event.html_url().unwrap()
-            ), config.meeting_label.clone())
+            (
+                format!(
+                    "A new proposal has been announced: [#{}]({}). It will be
+                announced at the next meeting to try and draw attention to it,
+                but usually MCPs are not discussed during triage meetings. If
+                you think this would benefit from discussion amongst the
+                compiler team, consider proposing a design meeting.",
```

Comment:
this mentions compiler team, but is this meant to be generic for both teams?

Reply:
Fixed, good catch. Also spotted another case of this (the team to ping on seconding) and added configuration for that (and pushed updates to lang and compiler team repos).